### PR TITLE
adding full task 1

### PR DESCRIPTION
**Pull request by KarolisTru**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+  width: 88%;
+  margin: 0 auto 50px auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(150px, 1fr));
+  gap: 20px 10px;
+}
+.product-card-regular {
+  justify-self: center;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  transition: 0.3s;
```
Comment (LukasTyla) [inline css/style.css:119]: It's common practice to specify what the transition will be applied to, in this case, it looks like it's `box-shadow`.

**Review by LukasTyla — COMMENTED** [review]
Looks great. 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+              <h3 class="product-name-carousel">Product Name</h3>
+              <p>
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut
+                commodo, magna ac semper malesuada, orci magna varius magna, vel
+                pharetra massa justo sed erat. Proin nec imperdiet erat.
+              </p>
+              <p>Price</p>
+            </div>
+          </section>
+          <button type="button">&rarr;</button>
```
Comment (visma-zanas-stundys) [inline index.html:132]: Instead of html entities, I'd recommend using pseudoelements with unicode chars ;)

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+  list-style-type: none;
+}
+
+footer ul li {
+  flex: 1 1 50%;
+}
+
+/*whole page media queries*/
+@media only screen and (min-width: 860px) {
+  .desktop-show {
```
Comment (visma-zanas-stundys) [inline css/style.css:164]: Consider splitting css into multiple files (utilities, components, etc...). This looks like an utility class. So maybe

```css
/* styles/utilities/display.css */
.desktop-show { display: none }
@media only screen and (min-width: 860px) { ... } 
```

You can use `@import` to add to `style.css` file ;)

**Review by visma-zanas-stundys — CHANGES_REQUESTED** [review]

**Pull request merged**